neuromation/api/config_factory.py

```diff
@@ -141,6 +146,25 @@ async def login_with_token(
         )
         self._save(config)
 
+    async def login_with_passed_config(
+        self,
+        config_data: str,
+        *,
+        timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT,
+    ) -> None:
+        try:
+            data = json.loads(config_data)
```

Comment:
Please work with base64-encoded json string, not bare json

Reply:
Done

Reply:
Why?